### ISSUE TITLE
Improve completion of text leading `alias`, `%`, `&` ...

### ISSUE THREAD

**Issue by scottming** [lexical-lsp/lexical#34]
```elixir
def print(s) do
  IO.puts(s)
end
```

`Enum.map(["a", "b"], &print|` should complete ``Enum.map(["a", "b"], &print/1)`

Currently, it gives:

```elixir
[{"detail":"Dummy.print/1(s)","insertText":"print($0)","insertTextFormat":2,"kind":3,"label":"print/1","sortText":"print/1"}]
```

**Comment by scottming** [comment]
<img width="340" alt="image" src="https://user-images.githubusercontent.com/12830256/234592488-1cd3aef6-0d90-4632-8f4d-d5f92a0aca11.png">

`&print|` still not complete `&print/1`, it completed `&print(name)`

**Comment by scohen** [comment]
Fixed by #145
